.github/workflows/build.yml

```diff
@@ -658,6 +658,17 @@ jobs:
       run: |
         cat howso_stacktrace.txt || true
 
+  init-devcontainers-release:
+    if: inputs.build-type == 'release'
+    needs: ["metadata", "build"]
+    uses: "howsoai/.github/.github/workflows/run-external.yml@main"
+    secrets: inherit
+    with:
+      repo: howso-devcontainers
+      run-only: true
```

Comment:
Why do you ignore the status checking here? Shouldn't we care here? I just want to make sure we surface when the devcontainer doesn't build. Maybe adding it to the release dashboard is enough if we really want to ignore the status here and not block a release.

Reply:
Updated